app.js

```diff
@@ -127,17 +141,5 @@ var s3 = new AWS.S3();
 
 startRequests();
 
-setInterval(function () {
-  var uptime = process.uptime();
-  console.log({
-    uptime: uptime,
-    listBufferSize: objectsToCopy.length,
-    succeeded: progress.succeeded,
-    failed: progress.failed,
-    gigabytesTransferred: progress.bytesTransferred/1024/1024/1024,
-    megabitsPerSecond: ((progress.bytesTransferred/1024/1024) * 8) / uptime,
-    objectsPerSecond: progress.succeeded / uptime,
-    marker: nextMarker
-  });
-
-}, 1000);
+setInterval(outputStatus, 1000).unref();
```

Comment:
is there only one instance of this app.js running? The reason I ask is the node docs state`In the case of setTimeout when you unref you create a separate timer that will wakeup the event loop, creating too many of these may adversely effect event loop performance -- use wisely.` https://nodejs.org/api/timers.html#timers_unref